src/utils/common.ts

```diff
@@ -89,3 +89,13 @@ export const buildUrlOptions = (options: { [key: string]: string }): string => {
   });
   return optionStr;
 };
+
+export const onCopy = async (valueToCopy: string, onSuccess?: () => void) => {
+  try {
+    await navigator.clipboard.writeText(valueToCopy);
+    alert('Copied!');
+    onSuccess?.();
```

Comment:
```suggestion
    if (onSuccess) onSuccess();
```

Reply:
Seemed fine to me and feels more concise, any reason to change this?